tests/roxctl/bats-tests/cluster/central-init-bundles.bats

```diff
@@ -13,15 +13,15 @@ setup_file() {
 }
 
 setup() {
-  output_dir="$(mktemp -d -u)"
+  output="$(mktemp -d -u)"
 }
 
 teardown() {
-  rm -rf "${output_dir}"
+  rm -rf "${output}"
 }
 
 @test "roxctl central init-bundles fetch-ca" {
-  run roxctl_authenticated central init-bundles fetch-ca --output ${output_dir}/ca-config.yaml
+  run roxctl_authenticated central init-bundles fetch-ca --output ${output}
```

Comment:
Does the `--output` accept a dir or a file path? If file, then we need this form: `${output}/ca-config.yaml` (also in assertion). WDYT?

Reply:
It accepts a file but the path must be valid (it does not create a dir)

Reply:
My original assumption was `mktemp` will give use a random name in temp dir but then it may be not clear if we accept dir or file. Let's remove `-u` and keep explicit file name